order-service/src/main/resources/application-dev.yml

```
@@ -1,6 +1,110 @@
+      promotionServiceClient:
+        base-config: default
+      paymentServiceClient:
+        base-config: default
+      couponServiceClient:
+        base-config: default
+
+order:
+  cancel-policy:
+    order-pending: 1
+  scheduler:
+    poll-interval: 30000
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)